src/pages/Borrow/BorrowIntro.vue

```diff
@@ -69,7 +69,15 @@
 				</kv-button>
 			</div>
 			<div class="small-6 columns">
-				<kv-button to="/borrow/pre-application/community" class="smaller cta-btn">
+				<kv-button
+					:to="{
+						path: '/borrow/pre-application/community',
+						query: {
+							refid: this.$route.query.refid
```

Comment:
`this` is implicit in templates, so this line should be `refid: $route.query.refid` (same for the button in `BorrowCommunity.vue`)